D/difmap/build_tarballs.jl

```diff
@@ -0,0 +1,48 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "difmap"
+version = v"2.5.5"
+
+sources = [
+    ArchiveSource("ftp://ftp.astro.caltech.edu/pub/difmap/difmap2.5e.tar.gz", "457cd77c146e22b5332403c19b29485388a863ec494fff87137176396fc6a9ff"),
+    ArchiveSource("ftp://ftp.astro.caltech.edu/pub/pgplot/pgplot5.2.tar.gz", "a5799ff719a510d84d26df4ae7409ae61fe66477e3f1e8820422a9a4727a5be4")
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir
+mkdir pgplot_build && cd pgplot_build/
+cat ../pgplot/drivers.list | sed 's|! PSDRIV|  PSDRIV|g' | sed 's|! GIDRIV|  GIDRIV|g' > drivers.list
+../pgplot/makemake ../pgplot/ linux g77_gcc
+sed -i 's|FCOMPL=g77|FCOMPL=gfortran|' makefile
+make && make clean
+cd ../uvf_difmap/
+sed -i 's|^PGPLOT_LIB="-lpgplot -lX11"|PGPLOT_LIB="-L/workspace/srcdir/pgplot_build -L/usr/X11R6/lib -Xlinker -R/workspace/srcdir/pgplot_build:/usr/X11R6/lib -lpgplot -lX11"|' configure
+sed -i 's|^USE_TECLA="1"|USE_TECLA="0"|' configure
+./configure linux-i486-gcc
+./makeall
+cp ./difmap $bindir
+cp ../pgplot_build/libpgplot.so $libdir
+"""
+
+platforms = [
```

Comment:
Given the set of platforms for which you're building, I suspect cross-compilation actually fails

Reply:
I just don't know how to compile it for other systems. The readme (linked above) only give command line arguments for linux.